board/samsung/aries/aries.c

```diff
@@ -255,6 +266,14 @@ int misc_init_r(void)
 {
 #ifdef CONFIG_ENV_VARS_UBOOT_RUNTIME_CONFIG
 	set_board_info();
+	env_set("fdtfile", board_linux_fdt_name[cur_board]);
+	switch(cur_board) {
+		case BOARD_GALAXYS:
```

Comment:
I'd sooner check for GALAXYS4G and FASCINATE4G here - they are the only ones without an MMC.

Reply:
Fixed.